SwatDB/SwatDBDataObject.php

```diff
@@ -1639,24 +1652,6 @@ public function unserialize(string $data): void
         ];
 
         foreach ($data as $property => $value) {
-            if ($value instanceof SwatDate && isset($value->year)) {
```

Comment:
This code was for migrating sessions using `PEAR_Date`. This is no longer necessary.